src/card-queue/components/CardQueue.js

```diff
@@ -114,18 +149,25 @@ export class CardQueue extends React.PureComponent {
     return <h1>There is no one around</h1>
   }
   render () {
-    const { current } = this.state
-    const { queue } = this.props
+    const {
+      queue,
+      current
+    } = this.props
     return (
       current < queue.size
       ? this.makeCard()
-      : (<EmptyCard callback={e => this.props.emptyCallback(e)} />)
+      : (
+        <EmptyCard callback={e => this.props.emptyCallback(e)} />
+      )
     )
   }
 }
 
 function mapStateToProps (state: AppState) {
   return {
+    profilePhoto: state.profile.profile.photos.first(),
```

Comment:
yes. 1st profile = profileRelated reducers, 2nd - the actual user profile info itsefl